### ISSUE TITLE
Step display text customisation

### ISSUE THREAD

**Issue by adamralph** [adamralph/xbehave.net#411]
_Outcome of #409._

**Comment by adamralph** [comment]
@eyalsk I just pushed 2.3.1 beta 1, which contains this feature.

Could you please give it a try? https://www.nuget.org/packages/Xbehave/2.3.1-beta0001-build733

You should be able to achieve what you want by doing something like this:

```c#
[assembly: BackgroundSuffixes]
public sealed class BackgroundSuffixesAttribute : Attribute, IFilter<IStepDefinition>
{
    public IEnumerable<IStepDefinition> Filter(IEnumerable<IStepDefinition> steps) =>
        steps.Select(step => step.DisplayText((string stepText, bool isBackgroundStep) =>
            stepText + (isBackgroundStep ? " (Background)" : null)));
}
```

**Comment by iam3yal** [comment]
@adamralph Excellent, thank you for these changes. 😉

**Comment by iam3yal** [comment]
@adamralph I've tested it just now and so far it looks good.

**Comment by adamralph** [comment]
Released in 2.3.1 - https://twitter.com/adamralph/status/998611366817746944

Thanks @eyalsk. Look out for your name in the release notes! https://github.com/xbehave/xbehave.net/releases/tag/2.3.1 🏆
